### PR TITLE
Staff Page Update

### PR DESCRIPTION
**Pull request by JoshHowker**
As requested by Master Rhodes

### CODE REVIEW

**Comment by atoff** [comment]
Thanks for this @JoshHawker - would you be happy to take a stab at doing the other few tasks listed here - https://vatsimuk.atlassian.net/browse/CORE-404?

Our DPO is Chris Pawley, and DCRM Sean Reedman

File: resources/views/site/staff.blade.php [hunk 1]
```diff
@@ -259,17 +255,25 @@ class="img-responsive center-block profile-picture"/>
...
-                        <h4>Operations Director (VATUK9)</h4><br/>
-                        <img src="{{ $teamPhotos[54] }}"
-                             width=50px
-                             class="img-responsive center-block profile-picture"/>
-                        <p class="text-center">Chris Pawley</p>
+                        <br/>
+                        <table class="table">
+                            <tbody>
+                                <tr>
+                                    <td>DCRM</td>
```
Comment (atoff) [inline resources/views/site/staff.blade.php]:
```suggestion
                                    <td>
                                    DCRM </br>
                                    (Division Conflict Resolution Manager)
                                    </td>
```

File: resources/views/site/staff.blade.php [hunk 2]
```diff
@@ -259,17 +255,25 @@ class="img-responsive center-block profile-picture"/>
...
+                                    <td>Sean Readman</td>
+                                </tr>
+                                <tr>
+                                    <td>DPO</td>
```
Comment (atoff) [inline resources/views/site/staff.blade.php]:
```suggestion
                                    <td>
                                    DPO </br>
                                    (Data Protection Officer)
                                    </td>
```

**Comment by JoshHowker** [comment]
@atoff changes you suggested have been added 👍

**Comment by atoff** [comment]
Thanks @JoshHowker

**Pull request merged**